src/sentry/static/sentry/app/views/organizationDetails.jsx

```diff
@@ -145,7 +147,10 @@ export default class OrganizationDetails extends Component {
   render() {
     return (
       <OrganizationContext includeSidebar {...this.props}>
-        <OrganizationDetailsBody>{this.props.children}</OrganizationDetailsBody>
+        <OrganizationDetailsBody>
+          <Hook name='component:sample-event-banner' />
```

Comment:
@adhiraj thoughts on this? It gives me the org context I need but hopefully without being too heavyhanded? If we decide to build out banner messaging we'll likely want something similar to `ModalStore` but for now this is fine.